configuration/apirunner.go

```diff
@@ -18,14 +18,15 @@ package configuration
 
 import (
 	"fmt"
+	"time"
 )
 
 // APIRunner holds configuration for api
 type APIRunner struct {
 	Address string
 	Call    string
 	RPC     string
-	Timeout uint32
+	Timeout time.Duration
```

Comment:
dimension of this value was seconds, now it microseconds. It will be misleading in configs.

Reply:
fix